src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './TodoList.scss';
+import { ToDo } from '../../types/types';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+type Props = {
+  todos: ToDo[],
+  setSelectedUserId: (arg0: number) => void,
+};
+
+export const TodoList: React.FC<Props> = ({ todos, setSelectedUserId }) => {
+  const [query, setQuery] = useState('');
+  const [completed, setCompleted] = useState('');
+
+  const isFiltered = () => {
+    if (completed === 'Active') {
+      return todos.filter(todo => (
+        todo.title.toLowerCase().startsWith(query.toLowerCase())
+        && todo.completed === false
+      ));
+    }
+
+    if (completed === 'Completed') {
+      return todos.filter(todo => (
+        todo.title.toLowerCase().startsWith(query.toLowerCase())
+        && todo.completed === true
+      ));
+    }
+
+    return todos.filter(todo => (
+      todo.title.toLowerCase().startsWith(query.toLowerCase())
+    ));
+  };
+
+  const filterd = isFiltered();
+
+  return (
+    <div className="TodoList">
+      <h2>Todos:</h2>
+
+      <div className="TodoList__list-container">
+        <form action="">
+          <input
+            type="text"
+            data-cy="filterByTitle"
+            value={query}
+            onChange={(event) => {
+              setQuery(event.target.value);
+            }}
+          />
+          <select
+            onChange={(event) => {
+              setCompleted(event.target.value);
+            }}
           >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
+            <option value="All">All</option>
+            <option value="Active">Active</option>
+            <option value="Completed">Completed</option>
```

Comment:
Створи та використовуй Enum
```suggestion
            <option value=Status.Active>Active</option>
            <option value=Status.Completed>Completed</option>
```